### ISSUE TITLE
Language in Geological Age

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#519]
emailed screenshot - Minimum and maximum time is not displaying in English

**Comment by bferguso** [comment]
Is this for all instances? I'm seeing english values on my browser.

**Comment by LeannePyle** [comment]
Yes in different browsers, but not everywhere, perhaps just with Epochs - 
Another example GSC Loc. 58209 shows "Sen ordovicium" under Minimum and Maximum time

**Comment by bferguso** [comment]
Ah - OK. It seems like the display value isn't english, but the LOV presented is. I'll have a look into this.

**Comment by bferguso** [comment]
This was caused by a bug in Arches core that only retrieves the first 10 values. If 'en' isn't in that first 10 it would display another language. Resolved this by deleting all languages except 'en','en-US' and 'fr'.

**Comment by bferguso** [comment]
Automated cleanup
